apps/prairielearn/elements/pl-matching/pl-matching.py

```diff
@@ -274,6 +281,14 @@ def render(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
     name = pl.get_string_attrib(element, "answers-name")
     display_statements, display_options = data["params"].get(name, ([], []))
+    options_placement_right = True
```

Comment:
You don't need to define this variable here, you already do it below.

Reply:
Oh, sorry about that. I've removed the redundant variable definitions now.